app/controllers/carts_controller.rb

```diff
@@ -1,44 +1,31 @@
 class CartsController < ApplicationController 
-    
-    def index
-        @orders= Order.of_user(current_user)
-        @carts= Cart.of_user(current_user)
-        @carts_notempty=Cart.find_by(user_id: current_user)
-        @user=current_user
-        @cart_totalcount=0;
-        @cost=0;
-        @c=1;
+ include MainHelper
+
+
+    def index      
     end
 
-    def create
-        
+    def create      
     end
     
     def show
-       id= params[:id]
-       menu=Menu.find(id)
-       cart_menuid=Cart.of_user(current_user).find_by(menu_id: menu.id)
+       menu=Menu.find(params[:id])
```

Comment:
menu=Menu.find(params[:id])
```suggestion
       menu = Menu.find(params[:id])
```

Reply:
changed